### PR TITLE
fix(text-input): fix bug about focus event & tab index

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skipping review is not a problem (```style```, ```chore``` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
  - [ ] These changes require a usage change
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [ ] Updated Storybook documents for component
- [ ] Wrote test codes for composable, utils and etc.
- [x] Tested with console(if usages are changed) 

### Description
SSIA

### CODE REVIEW

File: src/inputs/input/text-input/PTextInput.vue [hunk 1]
```diff
@@ -429,6 +429,9 @@ export default defineComponent<TextInputProps>({
                 initiateMenu();
             }
         };
+        const focus = () => {
+            inputRef.value?.focus();
+        };
```
Comment (piggggggggy) [inline src/inputs/input/text-input/PTextInput.vue:435]: question: Where is focus being used?

Comment (yuda110) [reply src/inputs/input/text-input/PTextInput.vue:435]: 👀

**Review by yuda110 — APPROVED** [review]
LGTM!
Is the `focus()` used in the console?
If so, I think a comment might be necessary.

File: src/inputs/input/text-input/PTextInput.vue (hunk 1 above)
Comment (sulmoJ) [reply src/inputs/input/text-input/PTextInput.vue:435]: `focus()` is used in console!

**Review by piggggggggy — APPROVED** [review]
LGTM!!

**Pull request merged**